packages/components/src/components/tabs/tab.jsx

```diff
@@ -23,6 +23,7 @@ const Tab = ({
     onClick,
     setActiveLineStyle,
     top,
+    children_count,
```

Comment:
Why is this still here?

Reply:
There is no way to take it out of the package. I need to check amount of tabs